apps/question-the-docs/backend/config.py

```diff
@@ -2,26 +2,27 @@
 
 
 class FastAPISettings(BaseSettings):
-    mongo_uri: str
+    mongo_uri: str = 'mongodb://localhost:27018/'
```

Comment:
How will we connect to the cloud DB here? Do we pass an `.env` file.

Reply:
You just change the URI.

Reply:
We could think about passing a password as an environment variable and formatting it into the `mongo_uri`

Reply:
This is probably me not using `pydantic` correctly, but for these kinds of things I don't provide a default value and instead set an environment variable, and `pydantic` seems to detect it. 

But this actually feels like a hack now that I think about it. Would be happy for anyone to tell me the best way to change these values at runtime with `pydantic`.